plugins/jobs/src/js/jobsDelete.js

```diff
@@ -0,0 +1,10 @@
+import StringUtil from "#SRC/js/utils/StringUtil";
```

Comment:
🎨 no `.ts` 😭 (this file)

Reply:
Also: shouldnt it be `jobDelete`? 🤔 For clarification I'd love to see these files name `JobsMenuDelete` or similar 🤔

Reply:
I think that does not work very well either. They export a single default function, which is the api i would like. They are not react components, so if I export JobsMenuDelete, I will be calling it like:

`JobsMenuDelete(params)`.

I think that this is a fine name for a file that exports a single function.